### ISSUE TITLE
Cannot read property 'value' of undefined

### ISSUE THREAD

**Issue by runely** [runely/calendar-homey#178]
New sentry logging uses values not yet defined...

```javascript
TypeError: Cannot read property 'value' of undefined
    at init (/lib/sentry-io.js:28:56)
    at IcalCalendar.onInit (/app.js:22:3)
    at /opt/homey-client/node_modules/homey-apps-sdk-v2/lib/SDK.js:216:31
    at FSReqCallback.oncomplete (fs.js:167:5)
```

**Comment by runely** [comment]
Values were only meant for additional data in sentry. Values removed.

**Comment by runely** [comment]
Fixed in release 0.4.4
